src/index.js

```diff
@@ -146,9 +152,9 @@ export default class Pipeliner {
 
   /**
    * Runs the specified pipeline, passing the arguments to each task.
-   * @param  {string}    pipeline the pipeline to run
+   * @param  {string}    pipeline The name of the pipeline to run
    * @param  {...object} args     Arguments to pass to the pipeline tasks
-   * @return {Promise}             A promise that is executed when the pipeline completes.
+   * @return {Promise}            A promise that is fulfilled when the pipeline completes; it is rejected if any task in the pipeline fails (throws an error or returns a promise that is rejected)
```

Comment:
Are we talking at cross-purposes? 

I fiddled around with mapSeries(), and if the mapper function throws an exception, mapSeries() must be catching it, because the promise returned by mapSeries() ends up rejected with that exception. So, in Pipliner.run(), if job() throws an exception, the promise returned by mapSeries() will reject with that exception.

On the other hand, if the job() returns a promise, any exception that happens "within" the promise (then() clauses and the like) just causes the promise to reject, and the promise returned by Promise.resolve() _follows_ that rejection – so this too will cause the promise returned by mapSeries() to reject.

That's the behavior I was at least trying to describe in my changes to the doc comment. I don't think there's a bug there.